src/whitelist.html

```diff
@@ -43,7 +43,6 @@
 <script src="lib/codemirror/addon/scroll/annotatescrollbar.js"></script>
 <script src="lib/codemirror/addon/search/searchcursor.js"></script>
 <script src="lib/codemirror/addon/selection/active-line.js"></script>
-<script src="lib/punycode.js"></script>
```

Comment:
I couldn't find anything here that uses this, but let me know if I've got it wrong.

Reply:
Seems it's not being used, but this makes me realize it should be used, as the entries in _"Trusted sites"_ should be rendered as Unicode, which is not the case currently. Can be removed until I fix this in the future.